internal/storage/db/sqlite3/sqlite3.go

```diff
@@ -69,12 +72,18 @@ func NewStore(ctx context.Context, conf *Conf) (*Store, error) {
 		return nil, fmt.Errorf("failed to migrate schema: %w", err)
 	}
 
-	storage, err := internal.NewDBStorage(ctx, goqu.New("sqlite3", db))
+	s, err := internal.NewDBStorage(ctx, goqu.New("sqlite3", db))
 	if err != nil {
 		return nil, err
 	}
 
-	return &Store{DBStorage: storage}, nil
+	if conf.Verify {
```

Comment:
The SQLite schema is automatically created by Cerbos so there's no need to verify it here.